### PR TITLE
CSS theme update

### PR DESCRIPTION
**Pull request by guywillis**
Amended CSS for use with version 2 theme
Added disabled text colour state
Added cross browser default styling override for select elements

### CODE REVIEW

**Comment by moloko** [comment]
+1

**Comment by oliverfoster** [comment]
+1

File: less/matching.less [hunk 1]
```diff
@@ -40,10 +37,14 @@
...
     border: none;
+    -webkit-border-radius: 0;
+	-moz-border-radius: 0;
+	border-radius: 0;
     background: transparent;
-    padding: @item-padding-top 0px @item-padding-bottom @item-padding-left;
+    padding: 15px 35px 15px 20px;
     cursor: pointer;
     color: @item-text-color;
+    width: 100%;
```
Comment (lc-thomasberger) [inline less/matching.less]: why 2 times width?

**Comment by zenduo** [comment]
+1

**Pull request merged**